### PR TITLE
global: fixed application bootstrapping

### PR DESCRIPTION
**Pull request by NRodriguezcuellar**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -66,7 +66,8 @@
     "invenio-jsonschemas>=1.1.1,<1.2.0",
     "invenio-oaiserver>=1.3.0",
     "invenio-pidstore>=1.2.2,<1.3.0",
-    "invenio-records-rest>=1.7.0,<1.8.0",
+    "invenio-records-rest>=1.8.0,<1.9.0",
+    "invenio-rest>=1.2.4",
```
Comment (zzacharo) [inline setup.py]: This is redundant I think :)

Comment (NRodriguezcuellar) [reply setup.py]: are you sure that invenio-rest 1.2.4 is bottom pinned somewhere?

File: setup.py [hunk 2]
```diff
@@ -66,7 +66,8 @@
     "invenio-jsonschemas>=1.1.1,<1.2.0",
     "invenio-oaiserver>=1.3.0",
     "invenio-pidstore>=1.2.2,<1.3.0",
-    "invenio-records-rest>=1.7.0,<1.8.0",
+    "invenio-records-rest>=1.9.0",
```
Comment (kpsherva) [inline setup.py]:
```suggestion
    "invenio-records-rest>=1.9.0,<1.10.0",
```

File: setup.py (hunk 1 above)
Comment (kpsherva) [reply setup.py]: invenio-rest is bottom pinned in invenio-records-rest, you can remove from here

Comment (NRodriguezcuellar) [reply setup.py]: alright :+1:

**Review by kpsherva — APPROVED** [review]

**Pull request merged**